.github/workflows/pr-lint-pr.yml

```diff
@@ -20,7 +17,36 @@ jobs:
     runs-on: ubuntu-latest
     name: Check linked issues
     steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - name: Checking non-content changes - 🔎
+        shell: pwsh
+        id: check_file_changed
+        run: |
+          # Diff HEAD with the previous commit
+          $diff = git diff --name-only HEAD^ HEAD
+
+          $SourceDiff = $diff | Where-Object { $_ -notmatch '^content/' -and $_ -notmatch '^public/' }
```

Comment:
can these directories be configurable in an environment variable so that it is easier to add/change more directories later on

Reply:
We can, but it will add more complexity, do you want me to spend some more time doing that?